reference/resources/hostendpoint.md

```diff
@@ -21,23 +21,33 @@ will use to apply
 to the interface.
 
+For `calicoctl` [commands]({{ site.baseurl }}/reference/calicoctl/overview) that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `hostendpoint`, `hostendpoints`, `hep`, `heps`.
+
 **Default behavior of external traffic to/from host**
 
-If a host endpoint is added and network policy is not in place, the {{ site.prodname }} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules). For host endpoints, {{ site.prodname }} blocks traffic only to/from interfaces that it’s been explicitly told about in network policy. Traffic to/from other interfaces is ignored.
+If a host endpoint is created and network policy is not in place, the {{site.prodname}} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules).
+For a named host endpoint (i.e. a host endpoint representing a specific interface), {{site.prodname}} blocks traffic only to/from the interface specified in the host endpoint. Traffic to/from other interfaces is ignored.
+For a wildcard host endpoint (i.e. a host endpoint representing all of a host's interfaces), {{site.prodname}} blocks traffic to/from _all_ interfaces on the host (except for traffic allowed by failsafe rules).
 
-> **Note**: Currently, for host endpoints with `interfaceName: *`, only [pre-DNAT
-> policy]({{ site.baseurl }}/reference/host-endpoints/pre-dnat) is
-> implemented.
-{: .alert .alert-info}
+However, profiles can be used in conjunction with host endpoints to modify default behavior of external traffic to/from the host in the absence of network policy.
+{{site.prodname}} provides a default profile resource named `projectcalico-default-allow` that consists of allow-all ingress and egress rules.
+Host endpoints with the `projectcalico-default-allow` profile attached will have "allow-all" semantics instead of "deny-all" in the absence of policy.
 
-For `calicoctl` [commands]({{ site.baseurl }}/reference/calicoctl/overview) that specify a resource type on the CLI, the following
-aliases are supported (all case insensitive): `hostendpoint`, `hostendpoints`, `hep`, `heps`.
+Note: If you have custom iptables rules, using host endpoints with allow-all rules (with no policies) will accept all traffic bypassing those rules.
+
+> Auto host endpoints specify the `projectcalico-default-allow` profile so they behave similary to pod workload endpoints.
+{: .alert .alert-info}
 
 > **Important**: When rendering security rules on other hosts, {{site.prodname}} uses the
 > `expectedIPs` field to resolve label selectors to IP addresses. If the `expectedIPs` field
 > is omitted then security rules that use labels will fail to match this endpoint.
 {: .alert .alert-danger}
 
+**Host to local workload traffic**
+
+Traffic from a host to its own workload is always allowed, despite any policy in place. This ensures that `kubelet` liveness and readiness probes always work.
```

Comment:
Suggest saying "workload endpoint (e.g. Kubernetes pod)" just so people don't confuse with other apps running (non-containerized) on the host.

Reply:
820f22c5fafc3a15a37fd21d8e164b513f5fba3e